frontend/src/app/libs/event-info/task/task.component.html

```diff
@@ -1 +1,49 @@
-<h3 *ngIf="task">{{ task.taskName }}</h3>
\ No newline at end of file
+<section class="app-task">
+    <h3 class="app-task__header" *ngIf="task">{{ task.taskName }}</h3>
+    <div class="app-task__question" [innerHTML]="task.question | safeHtml"></div>
+    <form
+        class="app-task__form"
+        action=""
+        [formGroup]="eventFormGroup">
+        <ng-container [ngSwitch]="task.taskType">
+            <ng-template *ngSwitchCase="'TEXT'" [ngTemplateOutlet]="textAnswer"></ng-template>
+            <ng-template *ngSwitchCase="'CODING'" [ngTemplateOutlet]="codeAnswer"></ng-template>
+            <ng-template *ngSwitchCase="'SINGLE'" [ngTemplateOutlet]="singleAnswer"></ng-template>
+            <ng-template *ngSwitchCase="'MULTIPLE'" [ngTemplateOutlet]="multipleAnswer"></ng-template>
+            <ng-template *ngSwitchDefault [ngTemplateOutlet]="textAnswer"></ng-template>
+        </ng-container>
+        <app-button
+            class="app-task__form-submit"
+            type="flat"
+            [color]="solution.status == 'DRAFT' ? 'primary' : 'accept'"
+            (click)="submitSolution()">
+            <ng-container [ngSwitch]="solution.status">
+                <ng-container *ngSwitchCase="'PENDING'">
+                    {{ "BUTTON.SENDED" | translate }}
+                </ng-container>
+                <ng-container *ngSwitchDefault>
+                    {{ "BUTTON.SEND" | translate }}
+                </ng-container>
+            </ng-container>
+        </app-button>
+    </form>
+</section>
+
+<ng-template #textAnswer [formGroup]="eventFormGroup">
+    <app-ckeditor
+        class="app-task__form-answer app-task__form-answer--text"
+        formControlName="answer"
+        placeHolder="{{ 'PLACEHOLDER.EVENT_TEXT_SOLUTION' | translate }}"
+        [value]="solution.status !== 'PENDING'"
```

Comment:
move = "..." to getter

Reply:
Correct me if I wrong ... you mean do smth like this:
formCOntrolName="getFormControlName"

class ... {
private FORM_CONTROLL_NAME = 'answer'
public get getFormControlName {
  return this.FORM_CONTROLL_NAME
}
}